library/Falcon/Connector/WordPress.php

```diff
@@ -152,7 +182,13 @@ protected function get_html_footer( $url ) {
 		return apply_filters( 'falcon.connector.wordpress.html_footer', $footer, $url );
 	}
 
-	protected function get_post_content_as_text( $post ) {
+	/**
+	 * Get text-formatted message for a post.
+	 *
+	 * @param WP_Post $post Post to notify for.
+	 * @return string Plain text message.
+	 */
+	protected function get_post_content_as_text( WP_Post $post ) {
```

Comment:
Why the type hint here, but not below?

Reply:
No real reason, just did it on a whim as I was going.